src/devices/legrand.ts

```diff
@@ -697,6 +697,52 @@ const definitions: DefinitionWithExtend[] = [
             await reporting.onOff(endpoint);
         },
     },
+    {
+        zigbeeModel: ['NLIS - Light switch'],
+        model: 'NLIS - Light switch',
+        vendor: 'Legrand',
+        description: 'Light switch 1 gang Legrand',
+        ota: ota.zigbeeOTA,
+        toZigbee: [tzLegrand.led_mode],
+        exposes: [eLegrand.ledInDark(), eLegrand.ledIfOn()],
+        extend: [deviceEndpoints({endpoints: {l1: 1}}), light({configureReporting: true, endpointNames: ['l1']})],
+    },
+    {
+        zigbeeModel: ['NLIS - Double light switch'],
+        model: 'NLIS - Double light switch',
+        vendor: 'Legrand',
+        description: 'Light switch 2 gang Legrand',
+        extend: [deviceEndpoints({endpoints: {l1: 1, l2: 2}}), light({configureReporting: true, endpointNames: ['l1', 'l2']})],
+    },
+    {
+        zigbeeModel: ['NLIS - Triple light switch'],
+        model: 'NLIS - Triple light switch',
+        vendor: 'Legrand',
+        description: 'Light switch 3 gang Legrand',
+        ota: ota.zigbeeOTA,
+        toZigbee: [tzLegrand.led_mode],
+        exposes: [eLegrand.ledInDark(), eLegrand.ledIfOn()],
+        extend: [deviceEndpoints({endpoints: {l1: 1, l2: 2, l3: 3}}), light({configureReporting: true, endpointNames: ['l1', 'l2', 'l3']})],
+    },
+    {
+        zigbeeModel: ['NLIV - Double shutter switch'],
+        model: 'NLIV - Double shutter switch',
+        vendor: 'Legrand',
+        description: 'Shutter switch 2 gang Legrand',
+        ota: ota.zigbeeOTA,
+        fromZigbee: [],
+        toZigbee: [tzLegrand.shutter_switch_multi_gang, tzLegrand.led_mode],
+        extend: [windowCovering({controls: ['lift', 'tilt']})],
```

Comment:
```suggestion
        extend: [windowCovering({controls: ['lift', 'tilt'], endpointNames: ['l1', 'l2']})],
```

Does this work instead of `tzLegrand.shutter_switch_multi_gang` and `.enum('shutter_state', ea.STATE_SET, ['OPEN', 'CLOSE', 'STOP']).withEndpoint('l1'),`